src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -48,6 +51,17 @@ function WorkspaceCategoriesPage({policyCategories, route}: WorkspaceCategoriesP
     const {translate} = useLocalize();
     const [selectedCategories, setSelectedCategories] = useState<Record<string, boolean>>({});
 
+    function fetchCategories() {
+        Policy.openPolicyCategoriesPage(route.params.policyID);
+    }
+
+    const {isOffline} = useNetwork({onReconnect: fetchCategories});
+
+    useEffect(() => {
+        fetchCategories();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
```suggestion
    useEffect(
        fetchCategories,
        // eslint-disable-next-line react-hooks/exhaustive-deps
        [],
    );
```

NAB. No need to create an anonymous function